build-configs/configs/BuildConfigType.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+import type { ThemeType } from '../themes/ThemeType'
+import type { FeatureFlagsType } from './featureFlags'
+
+export type BuildConfigType = {|
+  appTitle: string,
```

Comment:
Is it appTitle or appName? I feel we are inconsistent... Maybe lookup the change in locales.json

Reply:
Just copied this from the webapp build config which is kind of old, but I guess you are right. Will change that.